compiler/frontend/src/module/module.rs

```diff
@@ -17,28 +21,38 @@ pub enum ModuleKind {
 }
 
 impl Module {
-    pub fn from_package_root_and_file(
-        package_root: PathBuf,
-        file: PathBuf,
+    pub fn from_package_name(name: &str) -> Self {
+        Module {
+            package: Package::Managed(name.into()),
```

Comment:
If we're going to need an owned version of the name, we can accept that directly:

```suggestion
    pub fn from_package_name(name: String) -> Self {
        Module {
            package: Package::Managed(name),
```